sources/packages/api/src/app.module.ts

```diff
@@ -2,18 +2,26 @@ import { Module } from "@nestjs/common";
 import { AppController } from "./app.controller";
 import { AppService } from "./app.service";
 import { DatabaseModule } from "./database/database.module";
-import { ConfigController } from "./config/config.controller";
+import { ConfigController } from "./route-controllers/config/config.controller";
```

Comment:
Add config controller export in the route-controller/index.ts and then import the controller in line 7

Reply:
Goot catch! Code changed to import the ConfigController and ConfigService using their respective index.ts files.